deltachat-ios/Controller/AccountSetup/Instand Onboarding/InstantOnboardingViewController.swift

```diff
@@ -130,7 +130,7 @@ class InstantOnboardingViewController: UIViewController, ProgressAlertHandler {
 
     private func deleteProfileIconPressed(_ action: UIAlertAction) {
         dcContext.selfavatar = nil
-        contentView?.imageButton.setImage(UIImage(named: "person.crop.circle"), for: .normal)
+        contentView?.imageButton.setImage(UIImage(named: "camera"), for: .normal)
```

Comment:
ftr: unrelated change, this resets the avatar in the onboarding screen correctly